### PR TITLE
Refactoring: Timestamps

### PR DESCRIPTION
**Pull request by bobaaaaa**
From the FE perspective we only need two dates:

* publication date
* updated (Aktualisiert am)

Let's discuss!

### CODE REVIEW

**Review by thisismana — APPROVED** [review]
this is fine.

File: stroeer/core/v1/article.proto [hunk 1]
```diff
@@ -96,21 +96,12 @@ message Metadata {
...
-
-  // editorial date: Zuletzt publiziert
-  // see https://github.com/stroeer/tapir/issues/112
-  google.protobuf.Timestamp last_published = 5;
-
-  // editorial date: Erstellungsdatum
-  // see https://github.com/stroeer/tapir/issues/112
-  google.protobuf.Timestamp created = 6;
+  // editorial date: 'Publikationsdatum'
+  google.protobuf.Timestamp published = 4;
```
Comment (thisismana) [inline stroeer/core/v1/article.proto:100]:
```suggestion
  google.protobuf.Timestamp publish_time = 4;
```

File: stroeer/core/v1/article.proto [hunk 2]
```diff
@@ -96,21 +96,12 @@ message Metadata {
...
 
   // Manually set editorial date ('Aktualisierungsdatum') at which the article
   // was updated in seconds of UTC time since Unix epoch.
-  google.protobuf.Timestamp updated = 7;
+  google.protobuf.Timestamp updated = 5;
```
Comment (thisismana) [inline stroeer/core/v1/article.proto:104]:
```suggestion
  google.protobuf.Timestamp update_time = 5;
```

**Review by thisismana — CHANGES_REQUESTED** [review]

**Review by moritzzimmer — CHANGES_REQUESTED** [review]
@bobaaaaa we should adjust the names for `valid_from` and `to` as well to match naming recommendations

**Review by thisismana — APPROVED** [review]

**Review by moritzzimmer — APPROVED** [review]

**Pull request merged**